src/event.js

```diff
@@ -610,8 +610,7 @@ jQuery.event = {
 			event,
 			{
 				type: type,
-				isSimulated: true,
-				originalEvent: {}
```

Comment:
You shouldn't need the check. `delete` should still work fine even when the property doesn't exist. That said, it will always exist in this case.